src/peep/staff.c

```diff
@@ -155,6 +155,14 @@ static money32 staff_hire_new_staff_member(uint8 staff_type, uint8 flags, sint16
 		return MONEY32_UNDEFINED;
 	}
 
+	if (staff_type != STAFF_TYPE_HANDYMAN &&
+			staff_type != STAFF_TYPE_MECHANIC &&
+			staff_type !=	STAFF_TYPE_SECURITY &&
+			staff_type != STAFF_TYPE_ENTERTAINER)
+	{
+		return MONEY32_UNDEFINED;
+	}
+
```

Comment:
This segment, at least on github's view, has some funky spacing and linebreaking, not sure if it's intentional or even existant in code. Otherwise, i'm digging this PR.